src/content/en/shows/designer-vs-developer/podcast/DVDS1E01-creating-a-collaborative-environment.md

```diff
@@ -0,0 +1,40 @@
+project_path: /web/_project.yaml
+book_path: /web/shows/_book.yaml
+description: This week we speak to Jake Archibald about communication.
+
+{# wf_updated_on: 2017-03-07 #}
+{# wf_published_on: 2017-03-07 #}
+{# wf_podcast_audio: https://storage.googleapis.com/http-203-podcast/designer-vs-developer/dvd-s01-e01-creative-collaborations.mp3 #}
+{# wf_podcast_duration: 00:13:28 #}
+{# wf_podcast_fileSize: 12609607 #}
+{# wf_podcast_subtitle: This week we speak to Jake Archibald about communication. #}
+{# wf_featured_image: /web/shows/designer-vs-developer/podcast/images/dvd-series-cover.jpg #}
+
+# Creating a collaborative environment {: .page-title }
+
+Welcome to Designer Vs Developer, a show that tries to solve the challenges
+faced in industry by having an open conversation between the two, providing 
+take aways, solutions to workflows, tools & discussions on everyday struggles. 
+
+This week we speak to Jake Archibald about communication and the assumption
+that designers and developers sole role is to either make things look pretty
+or to be a cog in a machine with no soul.
+
+<img src="/web/shows/designer-vs-developer/podcast/images/dvd-s01-e01-cover.jpg
+" class="attempt-right">
+
+In this episode:
+
+* Roles and place in Design and Development
```

Comment:
'design and development' instead of 'Design and Developmen'. (Nouns aren't capitalized because they're important.